### PR TITLE
[Design Proposal] Setting proxy for Kaniko Pod

### PR DESCRIPTION
**Pull request by prary**
HTTP/HTTPs Proxy for kaniko pod while running in labs.

### CODE REVIEW

File: docs/design_proposals/kaniko-proxy-setting.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Title
+
+* Author(s): Prashant Arya
+* Design Shepherd:
```
Comment (tejal29) [inline docs/design_proposals/kaniko-proxy-setting.md]:
```suggestion
* Design Shepherd: Tejal Desai
```

File: docs/design_proposals/kaniko-proxy-setting.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  
+```yaml
+// ClusterDetails *beta* describes how to do an on-cluster build.
+type ClusterDetails struct {
+    
+    // http_proxy     
+    http_proxy string `yaml:"http_proxy,omitempty"`
+
+    // https_proxy
+    https_proxy string `yaml:"http_proxys,omitempty"`
```
Comment (tejal29) [inline docs/design_proposals/kaniko-proxy-setting.md]: hi @prary, are https_proxy and http_proxys the new config values?

Comment (tejal29) [reply docs/design_proposals/kaniko-proxy-setting.md]: Adding these 2 fields is not a big issue. I think the team is ok with it!
Would you be able to submit a PR?

Comment (prary) [reply docs/design_proposals/kaniko-proxy-setting.md]: Yes I make the changes and raise a PR for the same

Comment (prary) [reply docs/design_proposals/kaniko-proxy-setting.md]: Is there any deadline to raise PR?

**Comment by tejal29** [comment]
No there is no such deadline!

**Pull request merged**